projects/plugins/jetpack/extensions/blocks/payments-intro/edit.js

```diff
@@ -34,14 +40,33 @@ export default function JetpackPaymentsIntroEdit( { name, clientId, className }
 		}
 	} );
 
-	const renderVariationPicker = () => {
+	const displayVariations = usableVariations.length && registerBlockVariation;
+
+	let instructions = __( "Please select which kind of payment you'd like to add", 'jetpack' );
+	if ( hasPatterns ) {
+		instructions = __( 'Start by choosing one of our suggested layout patterns', 'jetpack' );
```

Comment:
Should we also add a final dot here to be consistent with the other instructions?
```suggestion
		instructions = __( 'Start by choosing one of our suggested layout patterns.', 'jetpack' );
```